src/main/java/bio/terra/service/snapshotbuilder/utils/QueryBuilderConstants.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.service.snapshotbuilder.utils;
+
+public final class QueryBuilderConstants {
+
+  private QueryBuilderConstants() {}
```

Comment:
I followed this article https://stackoverflow.com/questions/479565/how-do-you-define-a-class-of-constants-in-java and made a singleton file, thoughts on using an `interface`?

Reply:
I would split these constants into two groups:

- `SELECT` aliases (or column names) which must be matched by result readers after a query is executed
- tables and their fields, used internally by query code

For aliases, those are part of the external contract of the query code, so I would put them in `QueryBuilderFactory` which is the API layer between the service code and the query code generation. If we wanted to close things up a bit more, we avoid using these constants outside `QueryBuilderFactory` and require callers to call methods like `getCount(ResultSet rs)` which would hide the way column names are represented. These methods would be on the query builder that generates the query.

For tables and fields, I think we should try putting each table and its columns in a single class, something like
```
class ConceptTable {
   public static final String NAME = "concept";
   public static final String CONCEPT_NAME = "concept_name";
   public static final String CONCEPT_CODE = "concept_code";
...
}
```

Also regarding select aliases vs column names, we may want to change our queries to always use aliases instead of relying on matching column names in the result, which would further isolate the aliases from the column names in queries.

Reply:
In future refactoring, would we want to move functions into this class as well? Like functions that represent shared query building code